go.mod

```diff
@@ -203,3 +202,7 @@ require (
 	modernc.org/token v1.0.1 // indirect
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
+
+// We want to format raw bytes as hex instead of base64. The forked version
+// allows us to specify that as an option.
+replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
```

Comment:
Why not just import it directly?

Reply:
Because we need to keep it as the original package name, if I import it directly I get this error:
```
        module declares its path as: google.golang.org/protobuf
                but was required as: github.com/lightninglabs/protobuf-go-hex-display
```

And renaming the package in the fork would require a way bigger diff than just https://github.com/protocolbuffers/protobuf-go/commit/9c1f7c5b64c15ea4ffc186c309df4bf73680f1f2 as we'd need to refactor a bunch of usages of the `internal` package as well. So rebasing the hex render feature on top of a new version in the future would become a bigger task, potentially causing us to lag behind on the versions of this package.